packages/manager/modules/nutanix/src/dashboard/general-info/controller.js

```diff
@@ -30,10 +31,18 @@ export default class NutanixGeneralInfoCtrl {
   }
 
   getTechnicalDetails() {
+    this.loadingTechnicalDetails = true;
     return this.NutanixService.getClusterHardwareInfo(
       this.serviceInfo.serviceId,
       this.server.serviceId,
-    );
+    )
+      .then((technicalDetails) => {
+        this.clusterTechnicalDetails = technicalDetails.nutanixCluster;
+        return technicalDetails.baremetalServers;
+      })
+      .finally(() => {
+        this.loadingTechnicalDetails = false;
+      });
```

Comment:
no catch required for manage error case ?
if duable better into routing.

Reply:
No not required. Technical details are not shown in case of error.